measure/src/macros.rs

```diff
@@ -0,0 +1,112 @@
+/// macro that wraps the Measure::this function to make its usage more ergonomic
+///
+/// Use `measure!()` macro when you have a function that you want to measure.
```

Comment:
function -> expression, and add a newline
```suggestion
/// Use `measure!()` macro when you have an expression that you want to measure.
///
```